h/static/scripts/stream-controller.coffee

```diff
@@ -64,9 +60,9 @@ module.exports = class StreamController
       };
     );
 
+    annotationUI.setSortOptions(['Newest', 'Oldest'], 'Newest');
```

Comment:
You know, I've never noticed that you could sort the stream. It's a pretty useless feature, as it sorts the fetched cards, not the query. Perhaps consult with @conordelahunty on whether we even want to retain this? If not, we could get rid of all the extra code for changing the set of available sort keys.